crates/fiber-lib/src/rpc/dev.rs

```diff
@@ -102,6 +102,13 @@ pub struct SubmitCommitmentTransactionResult {
     pub tx_hash: Hash256,
 }
```

Comment:
seems this rpc is only used for e2e test, maybe you should use a smaller `CHANNEL_CHECK_SHUTDOWN_INTERVAL`, such as 6 seconds, and wait enough time in e2e test to make sure cron job is invoked automatically. and we can remove this rpc.